development/docker-compose.yml

```diff
@@ -20,6 +20,12 @@ services:
   worker:
     image: "networktocode/nautobot-dev-py${PYTHON_VER}:local"
     entrypoint: "nautobot-server rqworker"
+    healthcheck:
+      interval: 30s
+      timeout: 10s
+      start_period: 5s
+      retries: 3
+      test: ["CMD", "nautobot-server", "check"]
```

Comment:
Just wondering on this one - should this be calling `nautobot-server health_check` instead to use the command provided by django-health-check instead of Django's built-in command?